tests/test_ai_presenter.py

```diff
@@ -51,7 +50,7 @@ def testAiPresenter(self):
         # self.assertFalse(os.path.exists('text_ai.txt'))
         presenter.run()
         self.assertTrue(os.path.exists('text_ai.txt'))
```

Comment:
the os.path to look for should be ```blah.txt``` now bc that's the file name given in the yaml file